index.html

```diff
@@ -285,6 +285,22 @@ <h3 class="text-center text-dark">Our Instructors</h3>
         <div
           class="row row-cols-1 row-cols-md-2 g-2 mt-4 speaker-container"
         ></div>
+        <button
+          type="button"
+          class="btn btn-outline-dark btn-lg btn-block view-more"
+        >
+          View More
+
+          <i class="fa fa-arrow-down mx-2 text-danger" aria-hidden="true"></i>
+        </button>
+        <button
+          type="button"
+          class="btn btn-outline-dark btn-lg btn-block view-less"
+        >
+          View Less
+
+          <i class="fa fa-arrow-up mx-2 text-danger" aria-hidden="true"></i>
+        </button>
```

Comment:
- [ ] (OPTIONAL) Your speaker's section's display breaks as I change from desktop to mobile version. It would be great if you could fix that 👍 
- Your design:
![Animation](https://user-images.githubusercontent.com/85113567/168658157-e9478ac4-9dba-4738-9226-18b92b8ac0f9.gif)

Reply:
Thank you so much.


You procided me with a lot of ideas